File: payment-service/src/main/java/com/hedvig/paymentservice/services/payments/dto/PayoutMemberRequestDTO.java
package com.hedvig.paymentservice.services.payments.dto;

import com.hedvig.paymentservice.domain.payments.TransactionCategory;
import lombok.Value;

import javax.money.MonetaryAmount;

@Value
public class PayoutMemberRequestDTO {
  MonetaryAmount amount;
  TransactionCategory category;
  String referenceId;
  String note;
}

File: payment-service/src/main/java/com/hedvig/paymentservice/services/payments/dto/PayoutMemberRequestDTO.kt
package com.hedvig.paymentservice.services.payments.dto

import com.hedvig.paymentservice.domain.payments.TransactionCategory
import com.hedvig.paymentservice.domain.payments.enums.Carrier
import javax.money.MonetaryAmount

data class PayoutMemberRequestDTO(
    val amount: MonetaryAmount,
    val category: TransactionCategory,
    val referenceId: String?,
    val note: String?,
    val handler: String?,
    val carrier: Carrier?
)
